downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CreateGOAFile.java

```diff
@@ -0,0 +1,58 @@
+package org.reactome.release.downloadDirectory.GenerateGOAnnotationFile;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.*;
+
+public class CreateGOAFile {
+
+    private static final Logger logger = LogManager.getLogger();
+
+    private static final String CELLULAR_COMPONENT = "Cellular Component";
+    private static final String MOLECULAR_FUNCTION = "Molecular Function";
+    private static final String BIOLOGICAL_PROCESS = "Biological Process";
```

Comment:
It looks like these constants aren't used anymore.  Can they be removed?

Reply:
Done